projects/element-ng/menu/si-menu-item-checkbox.component.ts

```diff
@@ -27,6 +27,7 @@ import { SiMenuItemBase } from './si-menu-item-base.directive';
 export class SiMenuItemCheckboxComponent extends SiMenuItemBase {
   private cdkMenuItemCheckbox = inject(CdkMenuItemCheckbox);
   protected readonly icons = addIcons({ elementOk });
+  protected readonly hideCheckmark = input(false, { transform: booleanAttribute });
```

Comment:
No need to have an input. I think you can detect this automatically.
For normal menus consumer should never disable the checkmark.

`inject(CDK_MENU).orientation === 'horizontal'` --> menu bar.

Reply:
oh..I like that a lot better